dcos/http.py

```diff
@@ -97,10 +97,11 @@ def _request(method,
     except requests.exceptions.SSLError as e:
         logger.exception("HTTP SSL Error")
         msg = ("An SSL error occurred. To configure your SSL settings, "
-               "please run: `dcos config set core.ssl_verify <value>`")
+               "please run: `dcos config set core.ssl_verify <value>`, where "
+               "<value> is `True`, `False`, or a path to a CA_BUNDLE file.")
         description = config.get_property_description("core", "ssl_verify")
```

Comment:
@timcharper if we change the description here: https://github.com/dcos/dcos-cli/blob/3c95b2b80774d9d7b2888d7c88f4cdd42a405fe9/dcos/data/config-schema/core.json#L39 than we don't have to change this file at all :)